src/v/kafka/server/handlers/incremental_alter_configs.cc

```diff
@@ -353,6 +353,14 @@ create_topic_properties_update(
                   config::leaders_preference::parse);
                 continue;
             }
+            if (cfg.name == topic_property_cloud_topic_enabled) {
```

Comment:
This needs a test. It is pretty sensitive piece of code.

Reply:
yeh, good call. i'll add a test for this (and for the non-incremntal version as well). thanks!